automation/src/test/scala/org/broadinstitute/dsde/workbench/api/WorkbenchClient.scala

```diff
@@ -78,7 +78,7 @@ trait WorkbenchClient {
   }
 
   private def requestWithJsonContent(method: HttpMethod, uri: String, content: Any, httpHeaders: List[HttpHeader] = List())(implicit token: AuthToken): String = {
-    val req = HttpRequest(method, uri, List(makeAuthHeader(token)), HttpEntity(ContentTypes.`application/json`, mapper.writeValueAsString(content)))
+    val req = HttpRequest(method, uri, List(makeAuthHeader(token)) ++ httpHeaders, HttpEntity(ContentTypes.`application/json`, mapper.writeValueAsString(content)))
```

Comment:
fixed missing headers being passed along here. this should probably be fixed in our other copy of this code too

Reply:
Caught and fixed in other Swat 12/5.  We'll have to reconcile them carefully.

Reply:
Thanks. Isaac fixed this already in firecloud-ui on 12/5 which was before we started pulling code into workbench-libs which I also checked.